arcgis-ios-sdk-samples/Augmented reality/View hidden infrastructure in AR/ViewHiddenInfrastructureAR.storyboard

```diff
@@ -0,0 +1,179 @@
+                                <subviews>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text=" " textAlignment="center" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="G54-G0-CA5">
+                                        <rect key="frame" x="0.0" y="0.0" width="414" height="20.5"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                </subviews>
+                                <color key="backgroundColor" white="1" alpha="0.59999999999999998" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <constraints>
+                                    <constraint firstItem="G54-G0-CA5" firstAttribute="leading" secondItem="FTa-Ao-hiw" secondAttribute="leading" id="5pj-Mt-uwn"/>
+                                    <constraint firstItem="G54-G0-CA5" firstAttribute="top" secondItem="p3z-mu-aGJ" secondAttribute="top" id="rac-8x-xzn"/>
+                                    <constraint firstAttribute="bottom" secondItem="G54-G0-CA5" secondAttribute="bottom" id="sYT-pS-epi"/>
+                                    <constraint firstItem="G54-G0-CA5" firstAttribute="trailing" secondItem="FTa-Ao-hiw" secondAttribute="trailing" id="uDj-BF-VVl"/>
+                                </constraints>
+                                <viewLayoutGuide key="safeArea" id="FTa-Ao-hiw"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="C0j-oo-wc9">
+                                <rect key="frame" x="0.0" y="759" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem title="Calibrate" id="4HJ-ag-4mQ">
+                                        <connections>
+                                            <action selector="showCalibrationPopup:" destination="Ax7-UJ-W7h" id="9R1-PU-foW"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="MbD-L2-Ne8"/>
+                                    <barButtonItem style="plain" id="ymD-sU-NlE">
+                                        <segmentedControl key="customView" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="top" segmentControlStyle="bar" selectedSegmentIndex="0" id="lQs-at-RKk">
```

Comment:
I'm seeing a warning about this view being misplaced:

![image](https://user-images.githubusercontent.com/2257493/95298368-d16d8480-0830-11eb-97f8-35aefa9c0fce.png)

Are you seeing that?

Reply:
Hmm I saw it once, but after making some changes to the storyboard it disappeared.